README.md

```diff
@@ -18,12 +18,12 @@ Getting started with Service Modules is easy. Create a class that implements
 ```c#
 public class BugsnagModule : IServiceModule
 {
-    public void Configure(IServiceCollection services, IWebHostEnvironment env)
+    public void Configure(IServiceCollection services, IServiceConfigurationContext ctx)
```

Comment:
I used `ctx` throughout this PR but let me know if you prefer we spell this out as `context`.

Reply:
Considering we use env I would consider ctx to be in the same vein of understanding 😊